scrapers/SARJ-LLC.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+import sys
+import re
+import requests
+from urllib.parse import urlparse, urlunparse, urlencode
+
+def scrape_url(url, type):
+    parsed = urlparse(url)
+
+    path = parsed.path.split('/')
+    baseUrl = f"{parsed.scheme}://{parsed.netloc}"
+    if type == 'scene':
+        index = path.index('movie')
+        scraped = scrape_movie(baseUrl, path[index+1], path[index+2])
+    elif type == 'gallery':
+        index = path.index('gallery')
+        scraped = scrape_gallery(baseUrl, path[index+1], path[index+2])
+    elif type == 'performer':
+        index = path.index('model')
+        scraped = scrape_model(baseUrl, path[index+1])
+    else:
+        return None
+  
+    return scraped
+
+def search(fragment, type):
+    if type == 'scene' or type == 'gallery':
+      name = re.sub(r'\W', '_', fragment['title']).upper()
+      date = fragment['date'].replace('-', '')
+  
+      scraper = globals()['scrape_' + ('movie' if type == 'scene' else type)]
+      res = scraper('https://metartnetwork.com', date, name)
+      if res != None:
+          return res
+
+def fetch(baseUrl, type, arguments):
+    url =f"{baseUrl}/api/{type}?{urlencode(arguments)}"
+    response = requests.get(url)
+    if response.status_code >= 400:
+        return None
```

Comment:
Can you add a timeout for the `requests` get ?  Also a try for  exceptions should also be present eg network error. timeout ,... You can have a look at https://github.com/stashapp/CommunityScrapers/blob/1c65e2b793123713751cafe582745799a304b072/scrapers/vixenNetwork.py#L25 The referer and agent from there are not needed but don't hurt either.